langchain/memory/chat_message_histories/redis.py

```diff
@@ -21,6 +21,9 @@ def __init__(
         url: str = "redis://localhost:6379/0",
         key_prefix: str = "message_store:",
         ttl: Optional[int] = None,
+        db: int | None=...,
+        password: str | None=...,
```

Comment:
This should just be ``**kwargs`` and passed to the ``Redis.from_url`` method as such. There are a number of other things a user might want to pass like TLS settings. Either way, the username, password, and port should always just be passed in the url ``redis://username:password@host:port``